apps/codeforafrica/src/lib/api.netlify-cms/getTeam.js

```diff
@@ -1,15 +1,20 @@
 import { join } from "path";
 
 import getCollectionData from "./getCollectionData";
+import { setSeo } from "./seo";
 
 const teamDir = join(process.cwd(), "content/team");
 
 export default function getTeam(fields) {
   const teams = getCollectionData(teamDir, fields);
   return teams
     .filter((member) => !member.deactivated)
-    .map(({ slug = null, ...other }) => {
+    .map(({ slug = null, name, title, ...other }) => {
       const href = slug ? `/about/members/${slug}` : null;
-      return { ...other, slug, href };
+      const seo = setSeo({
+        title: name,
+        description: title,
+      });
+      return { ...other, slug, href, name, title, seo };
```

Comment:
SEO functionality should happen at _page_ level... not here. Probably should be in getPartnerPageStaticProps or something.